docs/examples/getting_started.md

```diff
@@ -68,16 +68,24 @@ cd ./example-get-started
 ```
 ### cmf init
 <pre>
-Usage: cmf init minioS3 [-h] --url [url] 
-                             --endpoint-url [endpoint_url]
-                             --access-key-id [access_key_id] 
-                             --secret-key [secret_key] 
-                             --git-remote-url[git_remote_url]  
-                             --cmf-server-url [cmf_server_url]
-                             --neo4j-user [neo4j_user]
-                             --neo4j-password [neo4j_password]
-                             --neo4j-uri [neo4j_uri]
+Usage: cmf init minioS3 [-h] --cmf-server-url [cmf_server_url] [required]
+                             --path [local_storage_path] [required]
+                             --git-remote-url[git_remote_url] [required]
+                             --url [url] [optional]
+                             --endpoint-url [endpoint_url] [optional]
```

Comment:
this is change is not correct. `--path` parameter is not needed for `cmf init minioS3`. And `--url` and `--endpoint-url` are required parameters. 
![cmf init minioS3](https://github.com/user-attachments/assets/613c0a02-e0fc-41c1-84a0-ed19c08f5d53)
Please check commands before making any changes in regards with usage of a command

Reply:
Got the issue with PR. Sorry for a bad PR. One question is there any specific reason why we are giving cmf init example with minioS3 instead of local in getting started?

Reply:
No worries Shreyas. There is no specific reason for choosing minioS3.  I guess, this was artifact repo we were testing extensively in the beginning when we were writing the commands and documentation. You can go ahead and change the example to `cmf init local`.

Reply:
I have changed it to local. Our thought was "cmf init minioS3" would overwhelm a new user. As user gets used to cmf user can switch to more advance options.